### PR TITLE
1353: Add a basic grade statistics popup

### PR DESCRIPTION
**Pull request by payten**
Delivers #1353 with a basic statistics summary, like this:

screen shot 2016-01-29 at 4 50 53 pm

I understand the issue describes some sort of graph representation... perhaps we can add that as a nice to have outside of this requirement, which may be to just show the hard data?

### CODE REVIEW

**Comment by payten** [comment]
Ah.. and just added "Out of", so it looks like this now:

screen shot 2016-01-29 at 4 58 20 pm

**Comment by payten** [comment]
Now with a "Done" button. Ok.. now I'm done :beer:

**Comment by steveswinsburg** [comment]
Ha, sounds like you had fun with it ;)

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeStatisticsPanel.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		super.onInitialize();
+
+		final Long assignmentId = ((Model<Long>) getDefaultModel()).getObject();
+
+		Assignment assignment = businessService.getAssignment(assignmentId.longValue());
+
+		add(new Label("title", new StringResourceModel("label.statistics.title",
+				null, new Object[] { assignment.getName() }).getString()));
+
+		List<GbStudentGradeInfo> gradeInfo = businessService.buildGradeMatrix(Arrays.asList(assignment));
```
Comment (steveswinsburg) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeStatisticsPanel.java]: Thinking about this some more, this method is probably a bit heavy to call for this purpose as it does all of the category/course grade calculations, sorting, permissions etc. We could probably build a cut down method for this purpose at some point.

**Pull request merged**